Add threshold value for 'periodSeconds' (#42159)
kubernetes/website: content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```

`periodSeconds` indicates the length of time in the past for which the policy must hold true.

> The default maximum value of `periodSeconds` is 1800 (half an hour).

Reviewer: ```suggestion
The maximum value that you can set for `periodSeconds` is 1800 (half an hour).
```

> The value of `periodSeconds` must be less than or equal to 1800 (half an hour).

Reviewer 2: ```suggestion
The default maximum value of `periodSeconds` is 1800 (half an hour).
```
Do we have a reason to state along with it, it will  further clarify the statement.
